type_templates/vrm.js

```diff
@@ -93,25 +180,53 @@ export default e => {
   const _cloneVrm = async () => {
     const vrm = await parseVrm(arrayBuffer, srcUrl);
     vrm.cloneVrm = _cloneVrm;
+    vrm.toonShaderify = _toonShaderify;
     return vrm;
   };
 
   const _prepVrm = (vrm) => {
-    //vrm.visible = false; //will need later
+    vrm.visible = false;

```

Comment:
Don't do this. If the VRM is invisible it is probably for good reason.

Reply:
there will always be at least 2 vrms loaded, the base(high quality) vrm, and the crunched vrm.  this defaults all the meshs to not visible to start, and the required meshes will be toggles accordingly.